wire-ios/Wire-iOS/Sources/Helpers/DynamicFontButton.swift

```diff
@@ -24,11 +24,20 @@ import WireCommonComponents
 class DynamicFontButton: StylableButton, DynamicTypeCapable {
 
     // MARK: - Properties
-    private let fontSpec: FontSpec
 
-    // MARK: - Initilization
+    private let onRedrawFont: () -> UIFont?
+
+    init(style: UIFont.FontStyle = .body) {
+        // Not needed when we use a font style.
+        onRedrawFont = { return nil }
+        super.init(frame: .zero)
+        self.titleLabel?.font = .font(for: style)
+        titleLabel?.adjustsFontForContentSizeCategory = true
+    }
+
+    @available(*, deprecated, message: "Use `init(style:)` instead")
     init(fontSpec: FontSpec = .normalRegularFont) {
```

Comment:
Should we already create a ticket to remove this deprecated initialize?

Reply:
Yes! I will create the ticket! Thank you for the reminder!